utils/swift_build_support/swift_build_support/products/product.py

```diff
@@ -31,12 +31,12 @@ def product_source_name(cls):
         return cls.product_name()
 
     @classmethod
-    def is_build_script_impl_product(cls):
-        """is_build_script_impl_product -> bool
+    def needs_toolchain(cls):
+        """needs_toolchain -> bool
 
-        Whether this product is produced by build-script-impl.
+        Whether this product needs a toolchain installed to be build.
```

Comment:
Nit-pick: This should probably read "... needs a toolchain installed to be built."